application/console/migrations/m170203_210542_create_initial_tables.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+use yii\db\Migration;
+
+class m170203_210542_create_initial_tables extends Migration
+{
+    public function up()
```

Comment:
If we use `safeUp()` and `safeDown()` (instead of simply `up()` and `down()`) then it will wrap these in a transaction, and Scrutinizer won't complain about method names being too short. :-)